code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -439,6 +439,15 @@ datum
 			on_mob_life(var/mob/M, var/mult = 1)
 				if(!M) M = holder.my_atom
 				M.stuttering += rand(0,2)
+				if(M.client)
+					for (var/obj/critter/domestic_bee/bee in view(7,M))
+						var/chat_text = null
+						var/text = pick_smart_string("shit_bees_say_when_youre_high.txt", "strings", list("M"="[M]", "beeMom"=bee.beeMom ? bee.beeMom : "Mom", "other_bee"=istype(bee, /obj/critter/domestic_bee/sea) ? "Seabees" : "Spacebees"))
```

Comment:
You already add the `s` at the end in the txt file. Also you probably also want to pass the "bee" argument which is this but reversed.